tests/runtests.py

```diff
@@ -588,7 +596,7 @@ def paired_tests(paired_test, options, test_labels, parallel, start_at, start_af
                 options.exclude_tags,
                 getattr(options, 'test_name_patterns', None),
                 options.start_at, options.start_after, options.pdb, options.buffer,
-                options.timing,
+                options.timing, options.shuffle,
```

Comment:
We should support `--shuffle` with `--bisect` and `--pair`, i.e. handle it in `get_subprocess_args()`.